test/helm-operator/cmd/helm-operator/main.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/helm/client"
+	"github.com/operator-framework/operator-sdk/pkg/helm/controller"
+	"github.com/operator-framework/operator-sdk/pkg/helm/release"
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+
+	"k8s.io/helm/pkg/storage"
+	"k8s.io/helm/pkg/storage/driver"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+var log = logf.Log.WithName("cmd")
+
+func printVersion() {
+	log.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
+	log.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
+	log.Info(fmt.Sprintf("operator-sdk Version: %v", sdkVersion.Version))
+}
+
+func main() {
+	flag.Parse()
```

Comment:
Do we have any defined flags that we're parsing?
The ansible-operator uses this for setting the reconcile/resync period.
https://github.com/water-hole/ansible-operator/blob/master/cmd/manager/main.go#L21

For helm we have that set to 5 seconds below. 
Slight tangent but I forget if we could configure that per CR.
https://github.com/water-hole/ansible-operator/blob/master/vendor/github.com/operator-framework/operator-sdk/pkg/ansible/controller/reconcile.go#L44-L47

Doesn't seem like we do that right now for helm, but I forgot if we're planning to do that.

Reply:
There are not flags defined by the helm operator itself, but there are flags defined in [github.com/golang/glog](https://github.com/golang/glog/blob/master/glog.go#L399-L404) and [sigs.k8s.io/controller-ruintime](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/config/config.go#L37-L43)

We don't have a command-line configurable or watches.yaml configurable reconcile periods, and I don't think we've talked about it for the Helm operator. We could plan to add that in a follow-on PR.

@shawn-hurley also mentioned dynamically adding watches for created objects. Not sure if we're thinking that will replace or augment the periodic reconciliation though.